src/main/app.js

```diff
@@ -1,7 +1,7 @@
 import { app, ipcMain as ipc } from 'electron';
 import pify from 'pify';
 import log from 'electron-log';
-import { throttle } from 'lodash';
+import debounce from 'lodash/debounce';
```

Comment:
I'd almost recommend [this one](https://www.npmjs.com/package/debounce), so that you wouldn't have to use lodash at all..

Reply:
But if it works it works..

Reply:
@perry-mitchell if we didnt use lodash anywhere else I would use this, maybe I'll remove lodash dep in the future as its really heavy